appveyor.yml

```diff
@@ -58,19 +65,19 @@ build_script:
   # examples
   - cd %ROOT_DIRECTORY%\examples
   - if defined bjam (
-    b2.exe --hash openssl-version=pre1.1 warnings=all warnings-as-errors=on -j %NUMBER_OF_PROCESSORS% %compiler% address-model=%model% picker-debugging=on invariant-checks=full variant=%variant% link=%linking% crypto=%crypto% asserts=on export-extra=on
+    b2.exe --hash openssl-version=pre1.1 warnings=all warnings-as-errors=on -j %NUMBER_OF_PROCESSORS% %compiler% address-model=%model% picker-debugging=on invariant-checks=full variant=%variant% link=%linking% crypto=%crypto% webtorrent=%webtorrent% asserts=on export-extra=on
```

Comment:
[here's](https://www.appveyor.com/docs/windows-images-software/) a list of what's available on the appveyor images. Since we're on the 2019 image, you could set:

`openssl-include=C:\OpenSSL-v111-Win64\include` and `openssl-lib=C:\OpenSSL-v111-Win64\lib`. I imagine that would work at least. Maybe this command line is getting long enough to warrant an environment variable, similar ot `$B2_ARGS` in `.travis.yml`

Reply:
you would most likely have to remove `openssl-version=pre1.1` as well, to pick up the new library names on windows

Reply:
Great, thank you!